packages/xo-server/src/api/host.js

```diff
@@ -2,6 +2,25 @@ import { format } from 'json-rpc-peer'
 
 // ===================================================================
 
+export function maintenanceMode({ host, maintenance }) {
```

Comment:
`toggleMaintenanceMode`?

Reply:
@pdonias `toggleMaintenanceMode` or `maintenanceMode` ?

Reply:
`setMaintenanceMode`? "toggle" would be if you wouldn't pass an argument and it would just switch on and off.